helmclient.go

```diff
@@ -143,13 +140,13 @@ func (c *Client) getReleaseHistory(ctx context.Context, releaseName string) (*Re
 }
 
 // InstallReleaseFromTarball installs a chart packaged in the given tarball.
-func (c *Client) InstallReleaseFromTarball(ctx context.Context, path, ns string, options ...helmclient.InstallOption) error {
+func (c *Client) InstallReleaseFromTarball(ctx context.Context, chartPath string, values map[string]interface{}, options InstallOptions) error {
```

Comment:
The values used to be an option. Now they are passed as a `map[string]interface{}`. `helmclient.MergeValues` already returns this so we are good.

https://github.com/giantswarm/helmclient/blob/31b69ea684193307663277dd36a69ed0e0ef3822/merge_values.go#L22